modules/identity-provider-registering-github.adoc

```diff
@@ -17,8 +17,8 @@ https://github.com/settings/applications/new[Register a new OAuth application].
 ** For GitHub Enterprise, go to your GitHub Enterprise home page and then click
 *Settings -> Developer settings -> Register a new application*.
 . Enter an application name, for example `My OpenShift Install`.
-. Enter a homepage URL, such as 
-`https://console-openshift-console.apps.example-openshift-cluster.com/`.
+. Enter a homepage URL, such as
+`\https://console-openshift-console.apps.<cluster-name>.<cluster-domain>/`.
```

Comment:
Should change to `https://oauth-openshift.apps.<cluster-name>.<cluster-domain>`

Reply:
Apologies; this has been corrected to use the indicated URL.